README.md

```diff
@@ -5,18 +5,19 @@ cancer dependencies and answer scientific questions about DepMap data.
 
 ## Quickstart
 
-[R](https://www.r-project.org/), install gfortran from
-https://github.com/fxcoudert/gfortran-for-macOS/releases for your version of OS.
-[conda](https://docs.conda.io/en/latest/miniconda.html),
-[Redis](https://redis.io/) (which can be installed by
-`brew install redis` if not already installed),
-[node.js](https://nodejs.org/en/).
-You may also need to
-`brew install pcre2` and `brew install gcc` and `brew install yarn`.
-
-If you are doing development on the "DepMap portal" web app, see the readme in [portal-backend](portal-backend).
-
-If you are doing development on a preprocessing job, see the readme in [pipeline](pipeline).
+The DepMap Portal codebase is a monorepo conisting of at least a few important modules:
```

Comment:
It was bothering me that the first instruction in this README was installing R (which we no longer use), so I wrote up a new "Quickstart" blurb. Any feedback is appreciated 😁